openedx/features/genplus_features/genplus_assessments/api/v1/serializers.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers
+
+from openedx.features.genplus_features.genplus.models import Student
+from openedx.features.genplus_features.genplus.models import Class
+from openedx.features.genplus_features.genplus_learning.models import (
+    ClassLesson,
+    ClassUnit
+)
+
+
+class ClassLessonSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = ClassLesson
+        fields = ('id', 'display_name', 'lms_url', 'usage_key')
+
+class ClassUnitSerializer(serializers.ModelSerializer):
```

Comment:
Most of these serializers are in the `genplus_learning/api/v1/serializers` section. Can we utilize that?

Reply:
I believe that we cannot utilize that serializer because the usage key and another field that is required in filter API are not defined in that serializer.

Reply:
You can always use inheritance in that case.

```
class NewSerializer(OldSerializer):
    class Meta:
         model = MyModel
         fields = BasicSerializer.Meta.fields + ('additional_field',)
```